### PR TITLE
Adds MongoDB nested fields documentation.

### PR DESCRIPTION
**Pull request by rvazquezglez**

### CODE REVIEW

File: docs/src/main/sphinx/connector/mongodb.rst [hunk 1]
```diff
@@ -244,6 +244,64 @@ Field           Required  Type      Description
 
 There is no limit on field descriptions for either key or message.
 
+.. note::
+
+    Trino can autogenerate the ``_schema`` contents by running a ``SELECT *`` over the collection.
+    However, not all the fields are always detected and you might need to update the schema manually.
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/mongodb.rst:250]: Similar sentence exists in `mongodb.schema-collection` section. How about fixing there? 
Also, I would like to mention `ALTER TABLE` statement approach in addition to manual `_schema` fix.

Comment (rvazquezglez) [reply docs/src/main/sphinx/connector/mongodb.rst]: Moved and mentioned `ALTER TABLE`.

File: docs/src/main/sphinx/connector/mongodb.rst [hunk 2]
```diff
@@ -77,6 +77,65 @@ At startup, this connector tries guessing fields' types, but it might not be cor
 
 This property is optional; the default is ``_schema``.
 
+.. note::
+
+    Trino can autogenerate the ``_schema`` contents by running a ``SELECT *`` over the collection.
+    However, not all the fields are always detected and you might need to update the schema manually or using ``ALTER TABLE``.
```
Comment (m57lyra) [inline docs/src/main/sphinx/connector/mongodb.rst:83]: What is the difference between updating the schema manually and using ALTER TABLE? Might want to explain.

File: docs/src/main/sphinx/connector/mongodb.rst [hunk 3]
```diff
@@ -77,6 +77,65 @@ At startup, this connector tries guessing fields' types, but it might not be cor
...
+        }
+        nesting_2_2: {
+          nested_field_2_2_1: <value>
+          nested_field_2_2_2: <value>
+          ...
+        }
+      }
+    }
+
+You can define the nesting by using ``row`` as follows:
```
Comment (m57lyra) [inline docs/src/main/sphinx/connector/mongodb.rst:114]: Is this self-describing in the same document or in another file? Might want to be a bit more explicit.

File: docs/src/main/sphinx/connector/mongodb.rst (hunk 2 above)
Comment (rvazquezglez) [reply docs/src/main/sphinx/connector/mongodb.rst:83]: With manually I meant updating directly on MongoDB, and `ALTER TABLE` is through Trino.

File: docs/src/main/sphinx/connector/mongodb.rst (hunk 3 above)
Comment (rvazquezglez) [reply docs/src/main/sphinx/connector/mongodb.rst:114]: It's actually an entry on the MongoDB schema collection.

File: docs/src/main/sphinx/connector/mongodb.rst (hunk 2 above)
Comment (m57lyra) [reply docs/src/main/sphinx/connector/mongodb.rst:83]: Great. Should be explicit about that for our users.

File: docs/src/main/sphinx/connector/mongodb.rst (hunk 3 above)
Comment (m57lyra) [reply docs/src/main/sphinx/connector/mongodb.rst:114]: Great. Should be explicit about that for our users. That's important for them to know.

File: docs/src/main/sphinx/connector/mongodb.rst (hunk 2 above)
Comment (rvazquezglez) [reply docs/src/main/sphinx/connector/mongodb.rst]: I updated it to be more explicit about the change.

File: docs/src/main/sphinx/connector/mongodb.rst (hunk 3 above)
Comment (rvazquezglez) [reply docs/src/main/sphinx/connector/mongodb.rst]: I updated it to be more explicit about the change.

File: docs/src/main/sphinx/connector/mongodb.rst [hunk 4]
```diff
@@ -80,7 +80,8 @@ This property is optional; the default is ``_schema``.
 .. note::
 
     Trino can autogenerate the ``_schema`` contents by running a ``SELECT *`` over the collection.
-    However, not all the fields are always detected and you might need to update the schema manually or using ``ALTER TABLE``.
+    However, not all the fields are always detected and you might need to update the schema directly on the MongoDB
```
Comment (m57lyra) [inline docs/src/main/sphinx/connector/mongodb.rst:83]: There are multiple issues in the passage. Suggest:

However, not all the fields are always detected. In this case, update the schema directly on the MongoDB
    entry, or through Trino using ``ALTER TABLE``."

The ALTER TABLE solution needs an example, as its use here is non-intuitive. I'd be specific with an example field that is missing, and how to construct the statement specifically to contend with that.

**Comment by colebow** [comment]
:wave: @rvazquezglez - this PR is inactive and doesn't seem to be under development. If you'd like to continue work on this at any point in the future, feel free to re-open.

**Pull request closed without merging**